apps/docs/src/guide/wallets/checking-balances.md

```diff
@@ -2,8 +2,14 @@
 
 To check the balance of a specific asset, you can use [`getBalance`](../../api/Account/Account.md#getbalance) method. This function aggregates the amounts of all unspent coins of the given asset in your wallet.
 
-<<< @/../../docs-snippets/src/guide/wallets/checking-balances.test.ts#checking-balances-1{ts:line-numbers}
+<<< @/../../docs-snippets2/src/wallets/checking-balances.ts#checking-balances-1{ts:line-numbers}
 
 To retrieve the balances of all assets in your wallet, use the [`getBalances`](../../api/Account/Account.md#getbalances) method, it returns an array of [`CoinQuantity`](../../api/Account/index.md#coinquantity). This is useful for getting a comprehensive view of your holdings.
 
-<<< @/../../docs-snippets/src/guide/wallets/checking-balances.test.ts#checking-balances-2{ts:line-numbers}
+<<< @/../../docs-snippets2/src/wallets/checking-balances-two.ts#checking-balances-2{ts:line-numbers}
+
+## Full Example
+
+For a full example of how to check balances, see the snippet below:
+
+<<< @/../../docs-snippets2/src/wallets/checking-balances.ts#full{ts:line-numbers}
```

Comment:
This documentation page includes two different code snippets:

- `checking-balances-1`
- `checking-balances-2`

Each snippet is independent, defined in its separate file with its own imports. I wonder if we need the full-example snippet here.

It seems the code from `checking-balances-2` was added to the `checking-balances-1` page just so it can be used within the full doc snippet example.

Reply:
1a7148913